docs/customisation/index.md

```diff
@@ -65,6 +65,16 @@ html_theme_options = {
 }
 ```
 
+### `announcement`
+
+Allows to show a site-wide announcement at the top.
```

Comment:
Adds a site-wide announcement, to the top of every page when set. This can contain HTML and is included as-is into the page.

Reply:
Changed, thanks!